### ISSUE TITLE
2.5.2 GetInstanceInfo() returns wrong difficultyID and difficultyName in Alterac Valley

### ISSUE THREAD

**Issue by Jinnai** [Stanzilla/WoWUIBugs#198]
In AV, GetInstanceInfo() returns:

```
[1]="Alterac Valley",
[2]="pvp", -- instanceType
[3]=2, -- difficultyID
[4]="Heroic",
...
```

In other BGs, GetInstanceInfo() returns:

```
...
[2]="pvp", -- instanceType
[3]=0, -- difficultyID
[4]="",
...
```

**Comment by Meorawr** [comment]
While it looks odd, does the difficulty data being wrong actually cause issues? I'd have thought that for battlegrounds those returns would be ignored.

**Comment by InfusOnWoW** [comment]
It does/did[1] causes issues in WeakAuras, because we treat the return values as orthogonal. 

[1] Can't remember whether we worked around it. I don't care that much about BCC.

**Comment by Jinnai** [comment]
> 
> 
> It does/did[1] causes issues in WeakAuras, because we treat the return values as orthogonal.
> 
> [1] Can't remember whether we worked around it. I don't care that much about BCC.

Yes, that's still the case. Instance Size Type load conditions work incorrectly with AV so auras for dungeons load in AV and auras for BGs don't. The issue was closed on WeakAuras issue tracker because it's a Blizzard API bug.
